.gitignore

```diff
@@ -5,3 +5,7 @@
 /var/
 /vendor/
 ###< symfony/framework-bundle ###
+
+/.idea
```

Comment:
This should only be in your `~/.gitignore_global`, not in the local `.gitignore`